scripts/_docker.sh

```diff
@@ -76,6 +78,65 @@ function _docker_build_with_custom_tag() {
     cd $root
 }
 
+function _docker_update_service() {
+    local account=$1
+    local env=$2
+    local service=$3
+
+    if [[ -z ${account} ]]; then
+        echo "Account is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${env} ]]; then
+        echo "Env is required" >&2
+        return 1
+    fi
+
+    if [[ -z ${service} ]]; then
+        echo "Service (back-end|ingestion|front-end) is required" >&2
+        return 1
+    fi
+
+    uhd docker ecr:login tools
+
+    local latest_image_tag
+    case "${service}" in
+        "back-end")
+            latest_image_tag=$(_docker_get_most_recent_back_end_image_tag)
+            ;;
+        "ingestion")
+            latest_image_tag=$(_docker_get_most_recent_ingestion_image_tag)
+            ;;
+        "front-end")
+            latest_image_tag=$(_docker_get_most_recent_front_end_image_tag)
+            ;;
+    esac
+
+    src_account_id=$(_get_tools_account_id)
+    dest_account_id=$(_get_target_aws_account_id $account)
+
+    local src_image="${src_account_id}.dkr.ecr.eu-west-2.amazonaws.com/ukhsa-data-dashboard/${service}:${latest_image_tag}"
+    local dest_image="${dest_account_id}.dkr.ecr.eu-west-2.amazonaws.com/uhd-${env}-${service}-"
+
+    if [[ "${service}" == "ingestion" ]]; then
+        dest_image+="lambda:${latest_image_tag}-${RANDOM}"
+    else
+        dest_image+="ecs:${latest_image_tag}-${RANDOM}"
+    fi
+
+    echo "Pulling ${src_image}..."
+    docker pull "${src_image}"
```

Comment:
add error handling:-
```suggestion
    docker pull "${src_image}" || { echo "Failed to pull image ${src_image}"; return 1; }
```